.teamcity/src/main/kotlin/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -70,15 +70,13 @@ fun BuildFeatures.publishBuildStatusToGithub(model: CIBuildModel) {
     }
 }
 
-fun BuildFeatures.triggeredOnPullRequests(model: CIBuildModel) {
+fun BuildFeatures.enablePullRequestFeature(model: CIBuildModel) {
     pullRequests {
         vcsRootExtId = "Gradle_Branches_GradlePersonalBranches"
         provider = github {
-            authType = token {
-                token = "%github.bot-gradle.token%"
-            }
-            filterAuthorRole = PullRequests.GitHubRoleFilter.MEMBER
-            filterTargetBranch = model.branch.branchFilter()
+            authType = vcsRoot()
+            filterAuthorRole = PullRequests.GitHubRoleFilter.EVERYBODY
+            filterTargetBranch = model.branch.pullRequestTargetBranchFilter()
```

Comment:
Can't you use `VersionedSettingsBranch.fromDslContext()` instead of passing the `model` around? I also wonder how important it is to filter the target branch.